src/ansible_navigator/utils/serialize.py

```diff
@@ -34,6 +82,17 @@ class JsonParams(NamedTuple):
     ensure_ascii: bool = False
 
 
+def _json_dump(dumpable: Dict, filename: Union[Path, str], file_mode: str):
+    """Create a file handle and dump json.
+
+    :param dumpable: The object to dump
+    :param filename: The file name for the file
+    :param file_mode: The file mode for the operation
+    """
+    with open(file=filename, mode=file_mode, encoding="utf-8") as file_handle:
```

Comment:
And having it always as a `Path`, you can call it's methods instead of having to work with low-level str-based APIs:
```suggestion
    with filename.open(mode=file_mode, encoding="utf-8") as file_handle:
```

Reply:
yep, nice.  Won't be able to do it for yaml yet.